TESTING.md

```diff
@@ -0,0 +1,34 @@
+Test
+---------------------------------
+
+This repository also provides a test framework which checks basic functionality
+of the Jenkins image.
+
+With v3, users can choose between testing Jenkins based on a RHEL (where you are running on a platform with subscriptions) or CentOS image.
+With v4, there is not CentOS vs. RHEL distinction, but we still use TARGET to control whether subscriptions are required when building the test image,
+and we reuse the v3 values (i.e. `rhel7`) for that purpose.
+
+*  **RHEL based image**
+
+    To test a RHEL7 based Jenkins image, you need to run the test on a properly
+    subscribed RHEL machine.
+
+    ```
+    $ cd jenkins
+    $ make test TARGET=rhel7 VERSION=2
```

Comment:
Are we still using rhel7 as the target, or it should be rhel8 now?

Reply:
updated in current push